apps/aecore/src/aec_conductor.erl

```diff
@@ -726,13 +740,13 @@ handle_block_candidate_reply({_Result, OldTopHash}, State)
     start_mining(State#state{block_candidate = undefined});
 handle_block_candidate_reply({Result,_OldTopHash}, State) ->
     case Result of
-        {ok, BlockCandidate, RandomNonce} ->
+        {ok, BlockCandidate, BlockCandidateTrees, RandomNonce} ->
```

Comment:
The BlockCandidateTrees should just be the root hash to avoid copying.

Again, this is out of scope for this ticket.

Reply:
Targeted by #513 (WIP).